packages/js/src/dashboard/index.js

```diff
@@ -110,6 +110,8 @@ export { Dashboard } from "./components/dashboard";
  * @property {CapabilitiesForSiteKit} capabilities The user capabilities for site kit.
  * @property {boolean} isVersionSupported Whether the current Site Kit plugin version is supported.
  * @property {boolean} isRedirectedFromSiteKit Whether the setup widget is shown after redirecting back from Site Kit.
+ * @property {string} storagePrefix The prefix for the client side cache key.
+ * @property {string} yoastVersion The Yoast version.
```

Comment:
Why are these in the Site Kit configuration? They are for the remote cached data provider? Which you added to the package (i.e. no reliance on site kit).

Reply:
Good point, moved them outside the site kit configuration. Done [here](https://github.com/Yoast/wordpress-seo/pull/22186/commits/4ea81ea66971ddc812975ab4b7a2ac56b2c601a1).